x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring/components/privileged_access_detection/info_popover.tsx

```diff
@@ -0,0 +1,124 @@
+import { useKibana, useNavigation } from '../../../../../common/lib/kibana';
+import { usePrivilegedAccessDetectionIntegration } from '../../../privileged_user_monitoring_onboarding/hooks/use_integrations';
+
+export const PrivilegedAccessInfoPopover = () => {
+  const { docLinks } = useKibana().services;
+  const { euiTheme } = useEuiTheme();
+  const [isPopoverOpen, { off: closePopover, toggle: togglePopover }] = useBoolean(false);
+  const entityAnalyticsLinks = docLinks.links.securitySolution.entityAnalytics;
+  const state = useIntegrationLinkState(ENTITY_ANALYTICS_PRIVILEGED_USER_MONITORING_PATH);
+
+  const { navigateTo } = useNavigation();
+  const padPackage = usePrivilegedAccessDetectionIntegration();
+  const navigateToPadIntegration = useCallback(() => {
+    navigateTo({
+      appId: INTEGRATION_APP_ID,
+      path: addPathParamToUrl(
+        `/detail/${padPackage?.name}-${padPackage?.version}/overview`,
+        ENTITY_ANALYTICS_PRIVILEGED_USER_MONITORING_PATH
+      ),
+      state,
+    });
+  }, [navigateTo, padPackage, state]);
+
+  const button = <EuiButtonIcon iconType="info" onClick={togglePopover} aria-label={'oi'} />;
+
+  return (
+    <EuiPopover
+      button={button}
+      isOpen={isPopoverOpen}
+      closePopover={closePopover}
+      className={css`
+        padding-left: ${euiTheme.size.s};
+      `}
+    >
+      <EuiPopoverTitle>
+        <EuiTitle size="xs">
+          <h4>
+            <FormattedMessage
+              id="xpack.securitySolution.entityAnalytics.privilegedAccessDetection.infoPopover.title"
+              defaultMessage="Top privileged access anomalies"
+            />
+          </h4>
+        </EuiTitle>
+      </EuiPopoverTitle>
+
+      <EuiText
+        size="s"
+        className={css`
+          max-width: 530px;
+        `}
+      >
+        <FormattedMessage
+          id="xpack.securitySolution.entityAnalytics.privilegedAccessDetection.infoPopover.description"
+          defaultMessage="{link} helps to detect anomalous privileged access activity in the Windows, Linux, and Okta system logs."
+          values={{
+            link: (
+              <EuiLink onClick={navigateToPadIntegration}>
+                <FormattedMessage
+                  id="xpack.securitySolution.entityAnalytics.privilegedAccessDetection.infoPopover.linkText"
+                  defaultMessage="The Privileged access detection ML package"
+                />
+              </EuiLink>
+            ),
+          }}
+        />
+        <p>
+          {
+            <FormattedMessage
+              id="xpack.securitySolution.entityAnalytics.privilegedAccessDetection.infoPopover.swimLanesDescription"
+              defaultMessage="Swim lanes provide an overview of the data buckets analyzed within the selected time period. A swim lane reflects the top 10 privileged users based on their anomaly scores, showing the maximum influencer score in each block."
```

Comment:
```suggestion
              defaultMessage="Swim lanes provide an overview of the data buckets analyzed within the selected time period. A swim lane reflects the top 10 privileged users based on their anomaly scores, showing the highest anomaly score for that user in each block."
```
Small update based on [this thread](https://docs.google.com/document/d/1j6zg3f736uAzYpdz6ldmKMSHHlAVwHEMan1y2Wc5NGU/edit?disco=AAABnaAN2AQ).